utility/src/test/java/org/oppia/android/util/networking/NetworkConnectionTestUtilTest.kt

```diff
@@ -55,56 +57,82 @@ class NetworkConnectionTestUtilTest {
   }
 
   @Test
-  fun testSetNetworkInfo_wifiShadowNetwork_connected_connectionStatusIsWifi() {
-    setNetworkConnectionStatus(
+  fun testSetNetworkInfo_wifiShadowNetwork_connected_returnsCorrectNetworkInfo() {
```

Comment:
Rather than 'returnsCorrectNetworkInfo' I think it helps to be specific like the names were before. One really useful property of test names is being able to understand what the test _intends_ to accomplish from the name alone. With the current name, we don't quite get that because we don't know what "correct network info" means or what to expect.

Reply:
Done.